packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -523,6 +524,14 @@ abstract class FlutterCommand extends Command<void> {
     );
   }
 
+  void addWasmCompileFlag() {
```

Comment:
This text is only accurate for the `build` command. There is no JS fallback for `run` or `test`. But this function is being used for all three commands.

Reply:
Oh! Ugh...k.

Reply:
Better @eyebrowsoffire ?